packages/grafana-data/src/dataframe/ArrowDataFrame.ts

```diff
@@ -173,6 +177,15 @@ export function grafanaDataFrameToArrowTable(data: DataFrame, keepOriginalNames?
   return table;
 }
 
+export function arrowTableToBase64String(table: Table): string {
```

Comment:
@kylebrandt or @ryantxu does this makes sense, basically needed just for testing so I can simulate a response from backend datasource so I can just mock a fetch response.

Reply:
LGTM -- although we are aiming to remove the base64 hack in grafana 8

Reply:
FYI -- the holdup and complexity of this is all based on the fact that everything gets munged though the TSDB backend object.  That is getting some attention and then we can have a simple frontend+backend shape that actually match!

Reply:
:tada: